sqlglot/tokens.py

```diff
@@ -166,7 +166,7 @@ def number(cls, number):
 
     @classmethod
     def string(cls, string):
-        return cls(TokenType.STRING, f"'{string}'")
+        return cls(TokenType.STRING, f"{string}")
```

Comment:
this should just be string no need to interpolate it if you're not adding quotes

Reply:
ah, good catch. I fixed quotes but forgot the interpolation